packages/itensor/quri_parts/itensor/load_itensor.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+from juliacall import Main as jl
+
+
+def ensure_itensor_loaded(calling_file_path: str) -> None:
```

Comment:
Rather than receiving `calling_file_path`, I think it is better to use the file name of this file.
If `__file__` is unavailable in the function, maybe you can capture it just above the function like `_FILE = __file__`.

Reply:
Or you can do the `library_path` resolution in the top level and do only the `include` in the function.